src/build123d/build_common.py

```diff
@@ -180,22 +189,30 @@ def _add_to_context(
     ):
         """Integrate a sequence of objects into existing builder object"""
         return NotImplementedError  # pragma: no cover
-
+    
     @classmethod
-    def _get_context(cls, caller=None):
-        """Return the instance of the current builder
+    def _get_context_or_raise(cls, caller=None) -> Self:
```

Comment:
Is there a reason to rename this method other than more lisibility ? I personally find it a bit weird to add `_or_raise` in the method name

Reply:
the reason is that in `__enter__` _get_context is used to set the parent with an Optional result, while in the rest of the codebase the _get_context function is used as if it could never return None